projects/fsastorefrontlib/src/cms-components/my-account/quote/quote-comparison/quote-comparison.component.html

```diff
@@ -0,0 +1,258 @@
+<ng-container *ngIf="quotesLoaded$ | async; else loading">
+  <div class="position-relative text-center">
+    <a
+      class="h4 vertical-center-left"
+      routerLink="/my-account/my-financial-applications/"
+      routerLinkActive="active"
+    >
+      <span class="fas fa-arrow-left"></span
+    ></a>
+    <h2 class="heading-headline p-3 mt-3 mb-5 text-center">
+      <span>
+        {{
+          'quote.quotesComparison'
+            | cxTranslate
+              : {
+                  category:
+                    'fscommon.category.' + categoryConfig?.categoryCode
+                    | cxTranslate
+                }
+        }}
+      </span>
+      <br />
+      <span>
+        {{ subheader }}
+      </span>
+    </h2>
+  </div>
+  <div class="row" *ngIf="quotes$ | async as quotes">
+    <div class="col-12">
+      <div class="accordion">
+        <div #firstAccordion>
```

Comment:
Can we put this reference on div with class accordion, instead of making a new empty div?